packages/@o3r/dynamic-content/tsconfig.fixture.jasmine.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
missing registration into the `tsconfig.json`'s references

Reply:
not done for all others fixture 
"Referenced project '/Users/crouzetm/git_clones/public-otter/packages/@o3r/dynamic-content/tsconfig.fixture.jasmine.json' must have setting "composite": true."
should I add composite true to all of them and reference them in their respectiver tsconfig ?